ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -534,22 +533,14 @@ def get_disk_safety(logger):
         points = []
 
         test_name = 'disk-safety'
-        result = {
-                "repair_percentage": None,
-                "lost_disks": None
-            }
 
-        abms = set(service.name.replace('arakoon-', '') for service in ServiceHelper.get_services()
-                   if service.type.name == ServiceType.SERVICE_TYPES.ALBA_MGR)
+        abm_services = set(service for service in ServiceHelper.get_services() if
+                           service.type.name == ServiceType.SERVICE_TYPES.ALBA_MGR)
 
-        abl = BackendHelper.get_albabackends()
-        for ab in abl:
+        for abm_service in abm_services:
+            alba_backend = BackendHelper.get_albabackend_by_guid(abm_service.abm_service.alba_backend_guid)
             # Determine if services are from ab instance
-            service_name = ServiceHelper.get_service(ab.abm_services[0].service_guid).name
-            if service_name not in abms:
-                continue
-
-            config = Configuration.get_configuration_path('ovs/arakoon/{0}/config'.format(service_name))
+            config = Configuration.get_configuration_path('ovs/arakoon/{0}-abm/config'.format(alba_backend.name))
```

Comment:
Yes, we are aware of cluster wide checks and will seperate these into a different addon in the future.